docs/articles/documentation/why-testcafe/README.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: docs
+title: Why TestCafe?
+permalink: /documentation/why-testcafe/
+---
+
+# Why TestCafe?
+
+TestCafe excels where other end-to-end testing solutions fall short.
+
+## Comprehensive and easy to set up
+
+TestCafe does not depend on Selenium or other testing software. TestCafe runs on the popular Node.js platform, and makes use of the browsers that you already have. For many users, the installation process consists of a single step:
+
+```js
+npm install -g testcafe
+```
+
+## Human-readable JavaScript tests
+
+TestCafe is a JavaScript framework. But you don’t need to be a JavaScript guru to write TestCafe tests. The core components of our API are intuitive and only require a basic knowledge of the language.
+
+## A first-class citizen in the Node.js ecosystem
+
+If you want to take your testing experience to the next level, you can always integrate additional JavaScript libraries and preprocessors into TestCafe tests.
+
+## Powerful but simple
+
+The advanced features of the TestCafe API can help you test complex, security-sensitive web applications.
+
+### Mock Requests
+
+Emulate HTTP responses to feed sample data to your app, troubleshoot connectivity errors, and cheat downtime.
+
+### Client-side injections
+
+Execute custom client-side code to parse the web page, examine its state, or even add extra dependencies.
+
+### Frames and windows
+
+Make use of multiple browser windows to test out complex user interactions. Switch to and from iframes with ease and stability.
+
+### Concurrent tests
+
+Run your tests in multiple browsers at once to quickly uncover browser-specific errors.
+
+## Automate what’s possible
+
+### Automated waiting
+
+Manual timeouts are a thing of the past. Asynchronous from the ground up, TestCafe knows when to wait and what to wait for before a test can continue.
+
+### Automated authentication scripts
+
+Save and activate user roles to easily switch between different user accounts.
+
+### Hooks
+
+Keep your code clean with hooks — routines that are repeated before and after each test.
+
+## Interactive when you need it
+
+### TestCafe Studio
+
+Use a GUI test recorder that makes the power of TestCafe accessible to people with zero programming experience.
+
+### Live mode
+
+Instantly see how the changes you introduce impact the test.
```

Comment:
```suggestion
Enable the Live Mode to instantly see how the changes you introduce impact the test.
```

Reply:
Again, now that I'm re-reading this, I don't understand what is being changed and what is being impacted. I change the page? Or I change the script? What reloads then?

Reply:
See https://github.com/DevExpress/testcafe/pull/5693/files#r532975785